fbw-a32nx/src/wasm/systems/a320_systems/src/hydraulic/sfcc/flaps_channel.rs

```diff
@@ -307,6 +456,10 @@ impl FlapsChannel {
     }
 
     fn flap_actual_position_word(&self) -> Arinc429Word<f64> {
+        if !self.is_powered && self.power_duration > Self::TRANSPARENCY_TIME {
```

Comment:
I think this condition is wrong?
What if `is_powered` is true (because it just got powered). Then it doesn't matter for how long it was powered it will not go into this branch. Or is this the intended behavior?